### ISSUE TITLE
Detect failure and provide suggestions

### ISSUE THREAD

**Issue by onoter** [gildas-lormeau/SingleFile#1213]
**Is your feature request related to a problem? Please describe.**
Saving pages in sspai like [this one](https://sspai.com/post/69945) will have formatting issues. After that happened, I did't know what caused the problem and tried other browsers and tried enabling and disabling different options, which caused a lot of headache. 

Luckily, this time I found the solution [here](https://github.com/gildas-lormeau/SingleFile/issues/1148#issuecomment-1497174724) in this repo. But I wish the extension could detect failures and auto suggest solutions. 

**Describe the solution you'd like**
I wish the extension could detect failure caused by hotlink protection(or anything else in general), and show the user what could be the problem and what could be the solution.

**Comment by gildas-lormeau** [comment]
Thanks for the suggestion. I have very mixed feelings about the feasibility of this feature. For example, it exists in "Save page WE", I tested it and I find it generates a lot of false positives, which is a pain. The problem is that web pages are full of errors of this type, but that doesn't prevent them from working properly. In other words, it seems complicated to distinguish between "normal" and "unexpected" errors.

BTW, this particular option will disappear in Chrome soon when I will be forced to migrate the extension to MV3. It means that images on sspai or weibo won't be saved anymore. It will continue to work in Firefox though.

**Comment by onoter** [comment]
Yeah, false positive will be a headache too. Overall, this kind of problem is quite rare using SingleFile. 

BTW, good to know it will continue to work in Firefox.
